server/src/main/java/com/objectcomputing/checkins/services/feedback_request/FeedbackRequestRepository.java

```diff
@@ -0,0 +1,22 @@
+package com.objectcomputing.checkins.services.feedback_request;
+
+import io.micronaut.data.annotation.Query;
+import io.micronaut.data.jdbc.annotation.JdbcRepository;
+import io.micronaut.data.model.query.builder.sql.Dialect;
+import io.micronaut.data.repository.CrudRepository;
+
+import javax.annotation.Nonnull;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.util.Set;
+import java.util.UUID;
+
+@JdbcRepository(dialect = Dialect.POSTGRES)
+public interface FeedbackRequestRepository extends CrudRepository<FeedbackRequest, UUID> {
+
+    @Override
+    <S extends FeedbackRequest> S save(@Valid @NotNull @Nonnull S entity);
+
+    @Query(value = "SELECT * from feedback_requests WHERE creatorId = :creatorId ")
```

Comment:
is there a reason we defined custom query here ?

Reply:
No, I have removed it and adjusted the call to `findByCreatorId` accordingly